editoast/src/views/infra/routes.rs

```diff
@@ -119,3 +128,78 @@ async fn get_routes_track_ranges<'a>(
 
     Ok(Json(result))
 }
+
+#[cfg(test)]
+mod tests {
+    use actix_http::StatusCode;
+    use actix_web::test::{call_service, read_body_json, TestRequest};
+    use rstest::rstest;
+
+    use crate::{
+        fixtures::tests::{db_pool, small_infra},
+        views::{
+            infra::routes::{WaypointResponse, WaypointType},
+            tests::create_test_service,
+        },
+    };
+
+    #[rstest]
+    async fn get_waypoint_should_return_da2_detector() {
```

Comment:
Not sure about the naming, this should not return a detector, but a list of routes from a detector.
Also I'm not a big fan of naming test function with an id inside. (here da2)

Reply:
Done.